espresso-support/core/src/main/java/com/novoda/espresso/ViewTestRule.java

```diff
@@ -25,11 +27,14 @@ protected Intent getActivityIntent() {
     }
 
     public void bindViewUsing(final Binder<T> binder) {
+        isIdle = false;
         getActivity().runOnUiThread(new Runnable() {
             @Override
             public void run() {
                 T view = getView();
                 binder.bind(view);
+                callback.onTransitionToIdle();
```

Comment:
I don't necessarily think this is wrong but I think you'd have a simpler life if you delegated to [`CountingIdlingResource`](https://developer.android.com/reference/android/support/test/espresso/idling/CountingIdlingResource.html)